src/layout/TextHighlight.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import * as F from 'futil-js'
+
+export default ({ pattern, text, Wrap = 'i' }) =>
+  pattern
+    ? F.highlight('<>', '<>', pattern, text)
+        .split('<>')
+        .map((x, i) => (i % 2 ? <Wrap key={i}>{x}</Wrap> : x))
```

Comment:
Is this wrapping in italics every other highlighted element? If so, why?

Reply:
Since we're using the same token for start and end and splitting on it, every even element in the matches would have been wrapped in the token and thus should be highlighted. It's also more efficient than writing a regex.